{"name": "test_Once_logged_in", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: '../config.json'", "trace": "self = <tests.main_test_home.TestHome testMethod=test_Once_logged_in>\n\n    def setUp(self):\n>       f = open('../config.json', 'r')\nE       FileNotFoundError: [Errno 2] No such file or directory: '../config.json'\n\ntests/main_test_home.py:11: FileNotFoundError"}, "start": 1681672202944, "stop": 1681672202944, "uuid": "ff39e7f3-f54c-4f66-87bb-8baee466b645", "historyId": "9f22f731e1011e438eca85065bf12129", "testCaseId": "e21070a7d8d888bbf399c986e7d2e9de", "fullName": "main.test_classes#test_Once_logged_in", "labels": [{"name": "suite", "value": "main"}, {"name": "subSuite", "value": "test_classes"}, {"name": "host", "value": "Siwars-MacBook-Pro.local"}, {"name": "thread", "value": "57983-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "main"}]}